pagopa/io-app
# feat: [IA-470] Paypal onboarding flow - part 1 of 2

## Short description
This PR implements the first part of the paypal onboarding flow.
The flow has been split in two parts to avoid huge PR

### paypal on boarding part 1
- start the onboarding flow
- show the start screen (some info about paypal)
- search for psp to associate to the paypal account
- store the selected psp

### video
https://user-images.githubusercontent.com/822471/142647184-77e994ca-e667-40d1-8fa0-3a89b2179e55.mov

### how to test
- use the dev-server
- make sure to have the [paypal FF turned on](https://github.com/pagopa/io-app/blob/master/.env.production#L59)
